src/generic/bmpsvg.cpp

```diff
@@ -218,6 +218,16 @@ wxBitmapBundle wxBitmapBundle::FromSVG(const char* data, const wxSize& sizeDef)
     return FromSVG(copy.data(), sizeDef);
 }
 
+/* static */
+wxBitmapBundle wxBitmapBundle::FromSVG(const unsigned char* data, size_t len, const wxSize& sizeDef)
+{
+    wxMemoryBuffer copy(len+1);
+    copy.AppendData(data, len);
+    copy.AppendByte(0);
+
+    return FromSVG((char*)copy.GetData(), sizeDef);
```

Comment:
I'd still use `wxCharBuffer` here, it looks like we could do it simply like this (but untested):
```suggestion
    wxCharBuffer copy(len);
    memcpy(copy.data(), data, len);

    return FromSVG(copy.data(), sizeDef);
```
and avoid any casts.